common/metrics/metric_defs.go

```diff
@@ -1563,6 +1563,12 @@ var (
 	ScheduleCancelWorkflowErrors                      = NewCounterDef("schedule_cancel_workflow_errors")
 	ScheduleTerminateWorkflowErrors                   = NewCounterDef("schedule_terminate_workflow_errors")
 
+	// Force replication
```

Comment:
Do we care about verification failure?

Reply:
i think we can use activity failure metrics. we can always add later if needed.